### PR TITLE
delete wget vagrant link, dead

### PR DESCRIPTION
**Pull request by KevinArtus**
Change vagrant install, wget vagrant link was obsolete.

### CODE REVIEW

File: en/installation_guide/install_with_vagrant.rst [hunk 1]
```diff
@@ -11,11 +11,11 @@ For developers, we provide a Vagrant-powered environment with provisioning so yo
 Install Vagrant
 ---------------
 The project is running on a Vagrant virtual environment built on VirtualBox to be production ready.
+Here is two differents ways to install it.
```
Comment (ngilain) [inline en/installation_guide/install_with_vagrant.rst]: I remember we have talked about not giving straight instructions, but more a link to the vagrant page. If I'm right, can you do that please ?

File: en/installation_guide/install_with_vagrant.rst~ [hunk 2]
```diff
@@ -0,0 +1,151 @@
+Install OpenOrchestra with vagrant
```
Comment (ngilain) [inline en/installation_guide/install_with_vagrant.rst~]: You must remove this temp file from your commit :)

**Comment by ngilain** [comment]
Think to report your modifications in install_with_vagrant_for_contributor

**Comment by KevinArtus** [comment]
@itkg-ngilain It's done, could you please check it ?

**Comment by ngilain** [comment]
:+1: for me, but as it's a PR of documentation, you need 2 validations before merging

**Comment by SOLenG** [comment]
The redirection to vagrant doc is a good choice : :+1:

**Pull request merged**